pephub/routers/api/v1/namespace.py

```diff
@@ -119,6 +120,7 @@ async def create_pep(
             p = Project(f"{dirpath}/{init_file.filename}")
             p.name = project_name
             p.description = description
+            p.pep_schema = pep_schema
```

Comment:
I think that we assign the schema in the `agent.project.create` function... At least that's what @khoroshevskyi said yesterday. See [these notes I took](https://github.com/pepkit/pephub/issues/137#issuecomment-1582958833)

Reply:
It looks like this:
```python
agent.create(
    ...
    pep_schema = 'random_schema_name'
    ...
)
```